doc/tutorials/objdetect/traincascade.markdown

```diff
@@ -288,12 +288,12 @@ Command line arguments of opencv_traincascade application grouped by purposes:
     -   -minHitRate \<min_hit_rate\>
 
         Minimal desired hit rate for each stage of the classifier. Overall hit rate may be estimated
-        as (min_hit_rate\^number_of_stages).
+        as (min_hit_rate ^ number_of_stages) @cite Viola04.
```

Comment:
Please add space between `Viola04` and `.` because some versions of Doxygen can interpret it incorrectly.